src/commands/admin/welcome.js

```diff
@@ -1,103 +1,101 @@
-const { SlashCommandBuilder } = require('@discordjs/builders');
-const { Permissions } = require('discord.js');
+import {SlashCommandBuilder} from "@discordjs/builders";
```

Comment:
You removed the Permissions here, it's used on L40

```
ReferenceError: Permissions is not defined
    at Module.execute (file:///D:/github/mkevenaar/NickNelson/src/commands/admin/welcome.js:40:45)
    at Client.<anonymous> (file:///D:/github/mkevenaar/NickNelson/src/nick-bot.js:53:27)
    at Client.emit (node:events:527:28)
    at InteractionCreateAction.handle (D:\github\mkevenaar\NickNelson\node_modules\discord.js\src\client\actions\InteractionCreate.js:74:12)
    at Object.module.exports [as INTERACTION_CREATE] (D:\github\mkevenaar\NickNelson\node_modules\discord.js\src\client\websocket\handlers\INTERACTION_CREATE.js:4:36)
    at WebSocketManager.handlePacket (D:\github\mkevenaar\NickNelson\node_modules\discord.js\src\client\websocket\WebSocketManager.js:351:31)
    at WebSocketShard.onPacket (D:\github\mkevenaar\NickNelson\node_modules\discord.js\src\client\websocket\WebSocketShard.js:444:22)
    at WebSocketShard.onMessage (D:\github\mkevenaar\NickNelson\node_modules\discord.js\src\client\websocket\WebSocketShard.js:301:10)
    at WebSocket.onMessage (D:\github\mkevenaar\NickNelson\node_modules\ws\lib\event-target.js:199:18)
    at WebSocket.emit (node:events:527:28)
```

Reply:
Should be fixed now!